<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CScadaUserVariableSelectScaleDialog</class>
 <widget class="QWidget" name="CScadaUserVariableSelectScaleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CScadaUserVariableSelectScaleDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="treeView"/>
     <widget class="QTableWidget" name="tableWidget"/>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="pHLaylout">
     <item>
      <widget class="QPushButton" name="pOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
